src/sql/workbench/services/jobManagement/common/jobManagementService.ts

```diff
@@ -15,6 +15,10 @@ export class JobManagementService implements IJobManagementService {
 	private _onDidChange = new Emitter<void>();
 	public readonly onDidChange: Event<void> = this._onDidChange.event;
 
+	// data oservables for steps tree
```

Comment:
spelling, but I'd also just remove this. No reason this needs to be tied to the tree specifically